x-pack/plugins/reporting/server/export_types/csv/generate_csv/hit_iterator.ts

```diff
@@ -68,11 +68,17 @@ export function createHitIterator(logger: LevelLogger) {
       );
     }
 
-    function clearScroll(scrollId: string | undefined) {
+    async function clearScroll(scrollId: string | undefined) {
       logger.debug('executing clearScroll request');
-      return callEndpoint('clearScroll', {
-        scrollId: [scrollId],
-      });
+      try {
```

Comment:
Should we wrap these in a try/catch, or do:

```js
return callEndpoint('clearScroll', { scrollId: [scrollId] }).catch((err) => {
        // Do not throw the error, as the job can still be completed successfully
        logger.warn('Scroll context can not be cleared!');
        logger.error(err);
});
```

Reply:
Just a matter of code/preference. The `.catch` is a bit more concise whereas the try/catch is more idiomatic async/await js

Reply:
It definitely could go either way, in which case I like to defer to the existing async style in other code in the file. In this case, async/await follows the code style in this file